devito/ops/operator.py

```diff
@@ -14,17 +15,19 @@ class OperatorOPS(Operator):
     """
 
     def __init__(self, *args, **kwargs):
+        self._ops_kernels = []
         super().__init__(*args, **kwargs)
 
     def _specialize_iet(self, iet, **kwargs):
-
         warning("The OPS backend is still work-in-progress")
 
         ops_init = Call(namespace['ops_init'], [0, 0, 2])
         ops_exit = Call(namespace['ops_exit'])
 
         dims = []
         for n, (section, trees) in enumerate(find_affine_trees(iet).items()):
+            ops_kernel = opsit(trees, n)
+            self._ops_kernels.append(ops_kernel)
```

Comment:
fuse?

Reply:
This will be changed in the future as `opsit` will return more than the kernel, so joining these lines is not needed